### PR TITLE
Site geo json query update

### PR DESCRIPTION
**Pull request by willmt80**
## Why

[Monday.com Ticket](https://code4community-team.monday.com/boards/925293934/views/18904377/pulses/1398350639)

No longer uses the scientific species name for a tree, instead uses the common name. Also removes the fields that have been removed from the map site query in this PR https://github.com/Code-4-Community/speak-for-the-trees-backend-v2/pull/61

## This PR

Literally just removed fields and changed 1 name in 1 object and changed the property fetched for the tree popup

## Screenshots

 

Capture

Notice all the new blue sites

## Verification Steps

 

I checked some trees popups to make sure their names were correct. Also for verification of the backend PR, I can see all of the new blue sites from inserting data earlier

### CODE REVIEW

File: src/components/mapPageComponents/ducks/types.ts [hunk 1]
```diff
@@ -63,15 +63,12 @@ interface SiteFeatureResponse {
 export interface SiteFeaturePropertiesResponse {
   id: number;
   treePresent: boolean;
-  diameter?: number;
-  species?: string;
-  updatedAt: string;
-  updatedBy: string;
-  address?: string;
```
Comment (SofieDunt) [inline src/components/mapPageComponents/ducks/types.ts:70]: 💯

File: src/components/mapPageComponents/mapView/index.tsx [hunk 2]
```diff
@@ -378,7 +378,7 @@ const MapView: React.FC<MapViewProps> = ({
           // Sets the information to display in the popup
           setActiveTreeInfo({
             id: siteId,
-            species: eventFeature.getProperty('species'),
+            species: eventFeature.getProperty('commonName'),
```
Comment (SofieDunt) [inline src/components/mapPageComponents/mapView/index.tsx:381]: Do you know why we use camelCase here and snake_case elsewhere (like `tree_present` and `block_id`)?

**Review by SofieDunt — APPROVED** [review]
Looks good!

File: src/components/mapPageComponents/mapView/index.tsx (hunk 2 above)
Comment (willmt80) [reply src/components/mapPageComponents/mapView/index.tsx]: There's no good reason for us to use snake case anywhere in the frontend

**Pull request merged**